rest-api-spec/api/indices.analyze.json

```diff
@@ -44,6 +44,18 @@
           "type" : "string",
           "description" : "The name of the tokenizer to use for the analysis"
         },
+        "detail": {
+          "type" : "boolean",
+          "description" : "With `true`, output each tokens within tokenizer chain with all token attributes . With `false`, (default: false)"
```

Comment:
May be changing a bit the description based on the comment I wrote about the guide?
Also s/output/outputs, s/tokens/token